src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -294,16 +290,25 @@ void GcsPlacementGroupManager::SchedulePendingPlacementGroups() {
 
   bool is_new_placement_group_scheduled = false;
   while (!pending_placement_groups_.empty() && !is_new_placement_group_scheduled) {
-    const auto placement_group = pending_placement_groups_.front();
-    pending_placement_groups_.pop_front();
+    auto iter = pending_placement_groups_.begin();
+    // Nothing to schedule
+    if (iter->first > absl::GetCurrentTimeNanos()) {
+      break;
```

Comment:
I am a little confused about how this will be retried. Is it retried by the Tick in this case? (If so can we comment it?)

Reply:
Yes, for this case, it'll be covered by Tick